packages/api/src/post-build.js

```diff
@@ -66,6 +92,19 @@ class PostBuildHandler {
         res.status(400).send({ error: err.message });
       }
     });
+
+    // route for parsing test input server side
+    this.app.post('/postbuildraw', async (req, res, next) => {
```

Comment:
I'd be tempted to call this just `POST /build`, if we end up having something other than a raw upload in the future, we can use the HTTP header `Content-Type` to specify the different types.